specification/implementation/metadata/general-requirements/index.md

```diff
@@ -1,66 +1,65 @@
-construct of Content Information Type Specifications to achieve an even higher level of interoperability.
+- navigate data and metadata components within the package (i.e. packaging metadata);
+- to validate that no component has been damaged during transfer or preservation (i.e. fixity information);
+- to understand the processes used when creating and managing the package (i.e. provenance and preservation metadata); and
+- to understand how the data within the package can be accessed (i.e. representation information).
 
-We implement the core metadata requirements with METS (Metadata Encoding & Transmission Standard, <http://www.loc.gov/standards/mets/>). In this specification we describe the core elements used, more elements are available in the METS standard and can be used in the own implementation.
+We regard descriptive metadata and most detailed technical metadata as outside the scope of the CSIP. As such, the CSIP does not aim to provide detailed semantic interoperability between different systems. However, as noted in Section 1.2, implementers are welcome to use the Content Information Type Specifications to achieve a  higher level of interoperability.
 
-Some of the core metadata requirements are already visible from the structure presented in the previous Section. As seen in the previous section one or more METS files can be present. The METS file describing the whole package is from now called "Root METS" and the METS file present in the Representation folder is called "Representation METS" in the rest of this document. The detailed specification of using METS within the CSIP is available in Section 5.3.
+We describe the core metadata requirements using the METS (Metadata Encoding & Transmission Standard, <http://www.loc.gov/standards/mets/>). We describe the core elements used, the METS standard describes other available elements which may be used in addition to those detailed in the CSIP.
 
-In addition to the METS files the CSIP recommends the inclusion of PREMIS metadata (PREservation Metadata Implementation Strategies, <http://www.loc.gov/standards/premis/>) in appropriate preservation metadata folders. This is especially relevant when aiming for an interoperable approach towards provenance and access to Information Packages. However, we recognise that, especially in the
+Some metadata requirements are implied by the structure presented in the previous Section. As seen, one or more METS files can be present in a package. The METS file describing the whole package is from now called "Root METS", or "Package METS" and the METS file in the Representation folders are called "Representation METS". The detailed specification of using METS within the CSIP is available in Section 5.3.
```

Comment:
Is there a reason why we need two names for the same thing?

Reply:
Sorry I dwelt on this as I (hope I?) misunderstood the subject as Rep level vs. Package level, but I think I now realise that you mean "Root METS" vs. "Package METS"? If that's the case then I prefer Package METS.

Reply:
I guess Phillip meant the synonyms:

>from now called "Root METS", or "Package METS"

I agree, we should tame the temptation to make the text more flowery, and instead, stick to one term per concept, wherever possible. I tend to favour "Root METS," but before strongly pushing it, I need to fully familiarise with the changes to structural rules, i.e. I'm afraid that if we allow IPs with no folder structure, "Root" would become meaningless.

Reply:
Yes sorry - I meant: 

> "Root METS", or "Package METS"

I also prefer package METS.